LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017-2023 The Ethereum Foundation
```

Comment:
Should leave the start year in place - the template is from 2019, but eth-utils is 2017

Reply:
Nice catch, thank you!